astroquery/simbad/core.py

```diff
@@ -309,10 +309,19 @@ def list_votable_fields(self):
 
         if not hasattr(self,'_votable_fields_table'):
             # load the table
-            votable_fields_table = Table.read(
-                get_pkg_data_filename(os.path.join('data',
-                                                   'votable_fields_table.txt')),
-                format='ascii')
+            dict_file = get_pkg_data_filename(
+                os.path.join('data', 'votable_fields_dict.json'))
+            with open(dict_file, "r") as f:
+                fields_dict = json.load(f)
+            rows_to_add = []
+            for num in range(0, len(fields_dict), 3):
+                rows_to_add.append(
+                    list(fields_dict.keys())[num:num + 3])
+            for i in range(3 - len(rows_to_add[-1])):
+                rows_to_add[-1].append('--')
+            votable_fields_table = Table(
```

Comment:
I don't see any point in keeping these in 3 columns, so what about simplifying it by simply printing out the keys? You also need to sort the output to be alphabetical.